src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultHybridCache.StampedeStateT.cs

```diff
@@ -14,7 +15,7 @@ internal partial class DefaultHybridCache
 {
     internal sealed class StampedeState<TState, T> : StampedeState
     {
-        private const HybridCacheEntryFlags FlagsDisableL1AndL2 = HybridCacheEntryFlags.DisableLocalCacheWrite | HybridCacheEntryFlags.DisableDistributedCacheWrite;
+        private const HybridCacheEntryFlags FlagsDisableL1AndL2Write = HybridCacheEntryFlags.DisableLocalCacheWrite | HybridCacheEntryFlags.DisableDistributedCacheWrite;
```

Comment:
I think at some point, there's always likely to be some terminology that is area-specific; arguably, using the long form every time *also* hurts readability, from the perspective of "I'm trying to express a nuanced thing in a brief way, so that I don't have absurdly long sentences to express a simple idea"; I think this is maybe an intractable problem, but I can give you a uniform resource locator to a hypertext transfer protocol (secure) page in hypertext markup language that you can access to... (etc etc)

I've added:

```
// note on terminology: L1 and L2 are, for brevity, used interchangeably with "local" and "distributed" cache, i.e. `IMemoryCache` and `IDistributedCache`
```

that do?